### DISCUSSION TITLE
Is there any way to receive pointer input but lock the pointer position?

### DISCUSSION THREAD

**Discussion by AlvinRey** [AvaloniaUI/Avalonia#15225] [Q&A]
Recently, I want to implement a dragable NumericUpDown control. I found that If I use the delta of pointers (current pointer position - last pointer position) as a basis for increase( or decrease), the Increasement ( or decreasement) will stop when the pointer reached the border of screen. So, I want to just receive pointer input, but do not move the pointer itself, is there any way to do that?

**Comment by maxkatz6** [comment]
Created a feature issue for that: https://github.com/AvaloniaUI/Avalonia/issues/15256
But no, it's not currently supported. You can use Pointer.Capture(), but it will work only until user releases a mouse button.
